js/index.js

```diff
@@ -1,2 +1,15 @@
-//import './index-require';
```

Comment:
I've removed the `require()` as WebPack doesn't load packages with `require()` just because the top level uses it. Instead it uses [`resolve.mainFields`](https://webpack.js.org/configuration/resolve/#resolvemainfields) to determine the load order.